frontend/app/dashboard/src/views/start/SetupStepRow.vue

```diff
@@ -44,13 +45,19 @@ const $saving = ref(false);
 
 enum Routes {
     Premises = 'gebouwen',
+    Groups = 'leeftijdstGroepen'
```

Comment:
De urls moeten altijd lowercase zijn, met dashes indien nodig

```suggestion
    Groups = 'leeftijdsgroepen'
```